deploy/chart/Chart.yaml

```
@@ -1,4 +1,4 @@
 apiVersion: v2
 name: spiceai
 description: Spice.ai OSS
-version: 0.1.32
+version: 0.1.33
```

```suggestion
version: 0.1.32
```
Chart version has been updated by the following PR today so we don't need to bump it IMO
https://github.com/spiceai/spiceai/commit/0cf35a06efef26db607cc8cfe6ea5e2dae6ce0ad